tests/e2e/lifecycle/mcg/test_mcg_namespace_s3_ops.py

```diff
@@ -226,17 +226,13 @@ def test_mcg_namespace_basic_s3_ops(
         last_key = get_list_and_verify(
             first_page_res, obj_keys[:mid_index], "Contents", version="v1"
         )
-        next_page_res = bucket_utils.s3_list_objects_v1(
-            s3_obj=mcg_obj,
-            bucketname=ns_bucket,
-            max_keys=max_keys,
-            marker=first_page_res["NextMarker"]
-            if platform == constants.AZURE_PLATFORM
-            else last_key,
-        )
-        get_list_and_verify(
-            next_page_res, obj_keys[mid_index:], "Contents", version="v1"
-        )
+        if not platform == constants.AZURE_PLATFORM:
```

Comment:
Please add some log message that this check is skipped when Azure platform is used. Something like:
```
else:
    log.warning('Skipping next page entries check for ListObjectV1 - not supported on Azure')
    log.warning('For more info see: https://bugzilla.redhat.com/show_bug.cgi?id=1917677')
```

Reply:
Done. Thank you.